src/doc/rustc/src/platform-support/riscv32-unknown-none-elf.md

```diff
@@ -1,12 +1,12 @@
-# `riscv32{i,im,ima,imc,imac,imafc}-unknown-none-elf`
+# `riscv32{i,im,ima,imc,imac,imafc,e,em,emc}-unknown-none-elf`
 
 **Tier: 2**
 
 Bare-metal target for RISC-V CPUs with the RV32I, RV32IM, RV32IMC, RV32IMAFC and RV32IMAC ISAs.
 
 **Tier: 3**
 
-Bare-metal target for RISC-V CPUs with the RV32IMA ISA.
+Bare-metal target for RISC-V CPUs with the RV32IMA, RV32E, RV32EM and RV32EMC ISAs.
 
 ## Target maintainers
```

Comment:
Did the RISC-V team agree to provide maintenance for the RV32E* targets?

Reply:
Oops, I did not mean to imply that 🙂 I'll remove the mention from this file.